Programming week 3: Move VS share to assignments

Tom-van-Woudenberg opened the pull request:
To be reviewed together with assignment: https://github.com/MUDE-2025/PA1.3/pull/2 and https://github.com/TUDelft-MUDE/TUDelft-MUDE.github.io/pull/6

includes changes from https://github.com/TUDelft-MUDE/book/pull/72

Tom-van-Woudenberg committed:
Add Markdown and file paths introduction docs

Added two new markdown files: one explaining file paths and best practices for cross-platform compatibility, and another introducing Markdown syntax, formatting, tables, and figures for programming reports and Jupyter notebooks.

Tom-van-Woudenberg committed:
Update _toc.yml

Tom-van-Woudenberg committed:
Update week_1_2.md

Tom-van-Woudenberg committed:
Update markdown.md

Tom-van-Woudenberg committed:
Merge branch 'simplify-programming-week-1' into programming_week_2

Tom-van-Woudenberg committed:
Update overview.md

Tom-van-Woudenberg committed:
Merge branch '2025-draft' into programming_week_2

Tom-van-Woudenberg committed:
Merge branch 'simplify-programming-week-1' into programming_week_2

Tom-van-Woudenberg committed:
Add credits

Tom-van-Woudenberg committed:
Merge branch 'simplify-programming-week-1' into programming_week_2

Tom-van-Woudenberg committed:
uddate credits

Tom-van-Woudenberg committed:
Move to assignments

Tom-van-Woudenberg committed:
Merge branch 'programming_week_2' into programming_week_3

Tom-van-Woudenberg committed:
Update overview.md

Tom-van-Woudenberg committed:
Merge branch 'simplify-programming-week-1' into programming_week_2

Tom-van-Woudenberg committed:
Merge branch 'programming_week_2' into programming_week_3

Tom-van-Woudenberg committed:
Merge branch '2025-draft' into programming_week_3

lkdmc commented:
I think all of the book is using this sentence

<img width="730" height="583" alt="image" src="https://github.com/user-attachments/assets/633aa10b-04b0-4e72-8daf-142e4099be8d" />

for example, (week_1_1_programming)= in this photo, am I allowed to delete these? It seems not necessary

lkdmc committed:
Programming week3 review

Almost same fix with week2, but there is no week_1_3.md file, is this intentional?

lkdmc commented:
Almost same fix with week2, but there is no week_1_3.md file, is this intentional?

lkdmc committed:
change back to gurobi from week1-1

Tom-van-Woudenberg committed:
Merge branch '2025-draft' into programming_week_3

Tom-van-Woudenberg committed:
Merge pull request #127 from lkdmc/programming_week_3

Programming week 3
